ActivityCodeGenerator重构后出现IllegalIntent时会Crash

```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.os.Parcelable android.os.Bundle.getParcelable(java.lang.String)' on a null object reference
        at com.tencent.shadow.core.loader.delegates.ShadowActivityDelegate.onCreate(ShadowActivityDelegate.kt:89)
        at com.tencent.shadow.core.runtime.container.GeneratedPluginContainerActivity.onCreate(GeneratedPluginContainerActivity.java:966)
        at com.tencent.shadow.core.runtime.container.PluginContainerActivity.onCreate(PluginContainerActivity.java:86)
        at android.app.Activity.performCreate(Activity.java:7136)
```

因为PluginContainerActivity和GeneratedPluginContainerActivity中的hostActivityDelegate是两个不同的域变量。出现IllegalIntent时只置空了PluginContainerActivity的hostActivityDelegate。

--- a/projects/sdk/core/activity-container/src/main/java/com/tencent/shadow/core/runtime/container/PluginContainerActivity.java
+++ b/projects/sdk/core/activity-container/src/main/java/com/tencent/shadow/core/runtime/container/PluginContainerActivity.java
@@ -75,6 +75,7 @@ public class PluginContainerActivity extends GeneratedPluginContainerActivity im
 
         boolean illegalIntent = isIllegalIntent(savedInstanceState);
         if (illegalIntent) {
+            super.hostActivityDelegate = null;
             hostActivityDelegate = null;
             Log.e(TAG, "illegalIntent savedInstanceState==" + savedInstanceState + " getIntent().getExtras()==" + getIntent().getExtras());
         }

